packages/contracts-bedrock/src/L1/SuperchainConfig.sol

```diff
@@ -0,0 +1,238 @@
+    /// @param data       Encoded update data.
+    event ConfigUpdate(UpdateType indexed updateType, bytes data);
+
+    /// @notice Semantic version.
+    /// @custom:semver 1.0.0
+    string public constant version = "1.0.0";
+
+    /// @notice Constructs the SuperchainConfig contract.
+    constructor() {
+        initialize({
+            _systemOwner: address(0),
+            _initiator: address(0),
+            _vetoer: address(0),
+            _guardian: address(0),
+            _delay: 0,
+            _sequencers: new T.SequencerKeys[](0)
+        });
+    }
+
+    /// @notice Initializer.
+    ///         The resource config must be set before the require check.
+    /// @param _systemOwner Owner of the contract.
+    /// @param _initiator   Address of the initiator who may initiate an upgrade or change to critical config values.
+    /// @param _vetoer      Address of the vetoer.
+    /// @param _guardian    Address of the guardian, can pause the OptimismPortal.
+    /// @param _delay       The delay time in seconds between when an upgrade is initiated and when it can be finalized.
+    /// @param _sequencers  The initial list of allowed sequencers
+    function initialize(
+        address _systemOwner,
+        address _initiator,
+        address _vetoer,
+        address _guardian,
+        uint256 _delay,
+        T.SequencerKeys[] memory _sequencers
+    )
+        public
+        reinitializer(2)
+    {
+        _setSystemOwner(_systemOwner);
+        _setInitiator(_initiator);
+        _setVetoer(_vetoer);
+        _setGuardian(_guardian);
+        _setDelay(_delay);
+
+        for (uint256 i = 0; i < _sequencers.length; i++) {
+            bytes32 sequencerHash = Hashing.hashSequencerKeys(_sequencers[i]);
+            allowedSequencers[sequencerHash] = true;
+            emit ConfigUpdate(UpdateType.ADD_SEQUENCER, abi.encode(_sequencers[i]));
+        }
+    }
+
+    /// @notice Getter for the systemOwner address.
+    function systemOwner() public view returns (address systemOwner_) {
+        systemOwner_ = Slot.getAddress(SYSTEM_OWNER_SLOT);
+    }
+
+    /// @notice Getter for the initiator address.
+    function initiator() public view returns (address initiator_) {
+        initiator_ = Slot.getAddress(INITIATOR_SLOT);
+    }
+
+    /// @notice Getter for the vetoer address.
+    function vetoer() public view returns (address vetoer_) {
+        vetoer_ = Slot.getAddress(VETOER_SLOT);
+    }
+
+    /// @notice Getter for the guardian address.
+    function guardian() public view returns (address guardian_) {
+        guardian_ = Slot.getAddress(GUARDIAN_SLOT);
+    }
+
+    /// @notice Getter for the delay address.
+    function delay() public view returns (uint256 delay_) {
+        delay_ = Slot.getUint(DELAY_SLOT);
+    }
+
+    /// @notice Getter for the paused address.
+    function paused() public view returns (bool paused_) {
+        paused_ = Slot.getUint(PAUSED_SLOT) > block.timestamp;
```

Comment:
Should this be `>` or `>=`?

Reply:
I think `>` because that way it is paused until the end up `duration`.